src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -385,23 +385,26 @@ static bool equalMetadataVectors(BSONObj& mdV1, BSONObj& mdV2)
 *
 * attributeValueAbsent -
 *
-* Check that the attribute doesn't have any value, taking into account its multi-valuated nature.
+* Check that the attribute doesn't have any value
 *
-* NOTE
-*   1. ValueTypeNumber and ValueTypeBoolean are always non-empty.
-*   2. In API version 2, the value of a ContextAttribute CANNOT be left out
+*/
+bool attributeValueAbsent(ContextAttribute* caP)
+{
+  return caP->valueType == ValueTypeNone;
```

Comment:
If we don't have yet a decission on this, I'd suggest to progress the PR as it is, then align this globally in all the code base. If you agree, mark this as NTC :)